x-pack/plugins/security_solution/public/common/components/sourcerer/index.test.tsx

```diff
@@ -64,12 +69,52 @@ describe('Sourcerer component', () => {
   const { storage } = createSecuritySolutionStorageMock();
   let store = createStore(state, SUB_PLUGINS_REDUCER, kibanaObservable, storage);
 
+  beforeAll(() => {
+    (useRouteSpy as jest.Mock).mockImplementation(() => [
+      { pageName: SecurityPageName.overview, detailName: undefined },
+    ]);
+  });
+
+  afterAll(() => {
+    (useRouteSpy as jest.Mock).mockImplementation(() => [
+      { pageName: SecurityPageName.overview, detailName: undefined },
+    ]);
```

Comment:
Why is it necessary? I think that setting the mock implementation inside `beforeAll` should be enough. 🤔

Reply:
yeah, we don't need this.